src/Generator/Generators/CSharp/CSharpCommentPrinter.cs

```diff
@@ -89,10 +89,17 @@ private static void GetCommentSections(this Comment comment, List<Section> secti
 
         private static string GetText(Comment comment, bool trim = false)
         {
+            System.Text.RegularExpressions.Regex regex =
```

Comment:
Do not copy the regex like this. Extract it to a public variable somewhere instead and reuse it.